aerogear/mobile-security-service-operator
# refactor(config-map): removed config map creation (AEROGEAR-9744)

https://issues.jboss.org/browse/AEROGEAR-9744

Removed the creation of a config map created for each MobileSecurityServiceApp CR as it is not needed.

## Verification

1. Replace the image in `operator.yaml` with `quay.io/ephelan/mobile-security-service-operator:AEROGEAR-9744`.
1. Deploy the operator and install the Mobile Security Service with `make install`.
2. Create an app with `make example-app/apply.`
3. Check for the app CR, eg: `oc get mobilesecurityserviceapp mobile-security-service-app`
4. Delete the app CR with `make example-app/delete`.
5. It should delete successfully.
